src/sharing/view/GroupSharingDialog.js

```diff
@@ -182,7 +182,9 @@ function showAddParticipantDialog(model: GroupSharingModel, texts: GroupSharingT
 			})
 			return buttonAttrs
 		}
-	}, locator.contactModel))
+		// we hardcode the maximum number of BTF suggestions so the suggestions dropdown does not overflow the dialog
+	}, locator.contactModel, 3)
```

Comment:
should use a constant here, avoid confusion later down the road (even with the comment)

Reply:
OK, I amended and pushed a commit.